### ISSUE TITLE
32 Sirens Documentation

### ISSUE THREAD

**Issue by BurnhamR** [MTDOJRP/docs#5]
We need to create a documentation page for SirenSetting Limit Adjuster 2.0.2 aka 32 Sirens Plugin.
Need to do a installation tutorial and download link.
- [Direct Download (cdn.skyy.cc)](https://cdn.skyy.cc/FiveM/SirenSetting_Limit_Adjuster_v2.zip)
- [Official Download and Page](https://www.lcpdfr.com/downloads/gta5mods/scripts/28560-sirensetting-limit-adjuster/)
- [Virus Total Scan](https://www.virustotal.com/gui/file/74fec47012be517504fb515abc913ece559f0ad680d0f16058ae6fb2ea46a9aa)

How to install:
1. Close FiveM if its open
2. Download the .zip file
3. Extract the .zip file.
4. Copy the ASI file to your FiveM Game data folder/plugins, Path should be `C:/Users/{your name}/AppData/Local/FiveM/FiveM Application Data/plugins`
5. Start your game and you are done!

What it does?
In simple terms, It allows our Emergency vehicles to have more and better lighting.
Advanced Definition:
This ASI modified your GTAV siren buffer limit allowing up to 32 lights per vehicle instead of the default 21 and additionally increases the amount of light object ID's allowed by FiveM natively from 255 to 65535, This allows for more custom lights for all the vehicles.

**Comment by Paneedah** [comment]
Added in commit [949feca](https://github.com/MTDOJRP/docs/commit/949feca5b837ab10c0141667a3363af3507ac432)
